core/web/chains_controller.go

```
@@ -91,6 +96,9 @@ func (cc *chainsController[I, C, R]) Create(c *gin.Context) {
 		return
 	}
 
+	chainj, _ := json.Marshal(chain)
```

Please do not suppress errors anywhere